scripts/handlers/progressbar-handle.js

```diff
@@ -1,14 +1,29 @@
 /* Get Our Elements (with querySelector)*/
+const progressBar = player.querySelector('.progress__filled');
+const ranges = player.querySelectorAll('.player__slider');
 
 
 
-/* when clicking on the progress bar,
-*  reaching that percentage of the video
+/* 
 *  (.currentTime - .duration - timeupdate event)
 */
-function handleProgressBar() { }
+function handleProgressBar() { 
 
+    const videoPercent = (video.currentTime / video.duration) * 100;
+   
+    progressBar.style.flexBasis = `${videoPercent}%`;
 
+}
 
 
-/* write your event listeners in event-listeners.js*/
\ No newline at end of file
+function handleRangeUpdate() {
+    video[this.name] = this.value;
+}
```

Comment:
Why did this function @Sayed94h, this function was completed already by @pavelbidenko2018 , can you delete this please ?

Reply:
First I did this because I thought that this was the progress bar and then when I deleted this then the range bar did not work and I wrote it back. Now, I am going to delete it again hahah

Reply:
Ok great 👍